usb/esp_tinyusb/tusb_tasks.c

```diff
@@ -12,6 +12,14 @@
 #include "tinyusb.h"
 #include "tusb_tasks.h"
 
+#if !CONFIG_TINYUSB_NO_DEFAULT_TASK
```

Comment:
Nitpick: it is sometimes hard to understand what is the goal when there are two or more denial in logic. 
I mean, here we have (NOT)*_NO_DEFAULT. 
Probably, there is a chance to change it to #if CONFIG_TINYUSB_DEFAULT_TASK.

Reply:
Thanks @roma-jam for the comment.
All the CONFIG options for background tinyusb task are removed now. Instead, runtime parameters are used.